starknet_py/tests/e2e/docs/guide/test_serializing.py

```diff
@@ -4,16 +4,28 @@
 from starknet_py.tests.e2e.fixtures.misc import read_contract
 
 
+def test_short_strings():
+    # docs-shortstring: start
+    from starknet_py.cairo.felt import decode_shortstring, encode_shortstring
+
+    # Convert a string literal to its felt value
+    encoded = encode_shortstring("myshortstring")
+    assert encoded == 0x6D7973686F7274737472696E67
+
+    # Decode a felt into a string
+    # Note that decoded value always has 31 characters in length.
+    # Extra `\x00` characters can be stripped to get only the encoded value.
```

Comment:
Stripped from the end maybe?

Reply:
Actually from the beginning. Updated the snippet used here so it shows how to strip `\x00`.